realsense2_description/urdf/_d435.urdf.xacro

```diff
@@ -14,6 +14,7 @@ aluminum peripherial evaluation case.
   <xacro:include filename="$(find realsense2_description)/urdf/_usb_plug.urdf.xacro" />
 
   <xacro:macro name="sensor_d435" params="parent *origin name:=camera use_nominal_extrinsics:=false">
+    <xacro:arg name="add_plug" default="false" />
```

Comment:
The other way to fix this is to add this arg like the others, see:

https://github.com/IntelRealSense/realsense-ros/blob/development/realsense2_description/urdf/_d415.urdf.xacro#L15

My suggestion is on the line above to do:

```
<xacro:macro name="sensor_d435" params="parent *origin  name:=camera  use_nominal_extrinsics:=false  add_plug:=false">
```

Reply:
Yes, it looks like it was just left off in this merge:

https://github.com/IntelRealSense/realsense-ros/commit/6a9cc994e3a9f63ddff1938a86ff04ca990d2551

It was added to the 415 macro:

https://github.com/IntelRealSense/realsense-ros/commit/6a9cc994e3a9f63ddff1938a86ff04ca990d2551#diff-27a1d25d2cf34294ff181fcac43ae14cR15

But not the 435 macro:

https://github.com/IntelRealSense/realsense-ros/commit/6a9cc994e3a9f63ddff1938a86ff04ca990d2551#diff-87e047b9db6df86f3e7f1ecf5fffa916R16